de.dlr.sc.virsat.project.ui/src/de/dlr/sc/virsat/project/ui/navigator/contentProvider/VirSatObjectFolderContentProvider.java

```diff
@@ -9,6 +9,10 @@
  *******************************************************************************/
 package de.dlr.sc.virsat.project.ui.navigator.contentProvider;
 
+import java.util.ArrayList;
+import java.util.List;

```

Comment:
I See this provider filters on hidden files following the unix convention with a "." at the beginning of the file. What i don't understand is the use in the context of the navigator?

Reply:
The provider existed already before this. I added the filtering in there to hide the generated .empty files. But in the newest commit this approach is now simplified:

Instead of modifying the content provider I registered the standard eclipse ".* resources" filter to the virsat navigator.